meshiphi/utils.py

```diff
@@ -12,6 +12,32 @@
 from calendar import monthrange
 from scipy.fftpack import fftshift
 
+def longitude_domain(long):
+    """
+    Converts any longitude degree value into one between -180:180
+    """
+    # Allow input type to be list or ndarray
+    if isinstance(long, list):
+        return [longitude_domain(x) for x in long]
+    elif isinstance(long, np.ndarray):
+        return np.array([longitude_domain(x) for x in long])
+    # Return same format as input at antimeridian
+    elif long in [-180, 180]:
+        return long
+    # Otherwise convert it to be within domain
+    else:    
+        return (long + 180) % 360 - 180
+
+def longitude_distance(long_a, long_b):
+    """
+    Calculates the angular distance between two longitude values
```

Comment:
As I understand it this always calculates the distance travelling West to East from long_a to long_b (not the shortest distance). It would be good to make that explicit in the docstring.